lib/pltr/v2/reducers/__tests__/root.test.js

```diff
@@ -0,0 +1,56 @@
+import { ADD_LINES_FROM_TEMPLATE } from '../../constants/ActionTypes'
+import rootReducerWithoutRepairers from '../root'
+import { beatsByPosition } from '../../helpers/beats'
+
+const rootReducer = rootReducerWithoutRepairers({
+  normalizeRCEContent: (x) => x,
+})
+
+describe('rootReducer', () => {
+  describe('ADD_LINES_FROM_TEMPLATE', () => {
+    const enoughBeats = require('./fixtures/enough_beats.json')
+    const notEnoughBeats = require('./fixtures/not_enough_beats.json')
+    const templateData = require('./fixtures/7_point_template.json')
+    const action = { type: ADD_LINES_FROM_TEMPLATE, templateData }
+    describe('enough beats', () => {
+      const result = rootReducer(enoughBeats, action)
+      const beatsAfter = beatsByPosition(() => true)(result.beats['1']).map(({ id }) => id)
+      it('doesnt add more beats', () => {
+        const beatsBefore = beatsByPosition(() => true)(enoughBeats.beats['1']).map(({ id }) => id)
+        expect(beatsBefore.length).toEqual(beatsAfter.length)
```

Comment:
BTW, `expect` has a better way of checking array equality: https://jestjs.io/docs/expect#expectarraycontainingarray
(Not that I'm asking you to change the test because this provides good enough validation.)